probeinterface/io.py

```diff
@@ -14,7 +14,7 @@
 import json
 from collections import OrderedDict
 from packaging.version import Version, parse
-
+import pandas as pd
```

Comment:
pandas is not a hard depency of porbeinterface but an optional one.
The best to to import pandas in the functions.